src/DevTools/renderRelayTree.tsx

```diff
@@ -85,20 +85,23 @@ export function renderRelayTree<
   params: MockRelayRendererProps & {
     renderUntil?: RenderUntilCallback<P, S, C>
     wrapper?: (renderer: JSX.Element) => JSX.Element
+    variables?: object
   }
 ) {
   const {
     Component,
     query,
     mockResolvers,
     renderUntil: renderUntilCallback,
+    variables,
     wrapper,
   } = params
   const renderer = (
     <MockRelayRenderer
       Component={Component}
       mockResolvers={mockResolvers}
       query={query}
+      variables={variables}
```

Comment:
@alloy - noticed that this wasn't being passed through which made it impossible to pass variables when not using the `MockRelayRender`.

Reply:
After having skimmed the full diff, I think hardcoding the artist ID would have worked too 🤷‍♂️

Reply:
I feel like providing the ability to pass `variables` maps better to the mental model when writing tests for existing code, like one sees things like 

https://github.com/artsy/reaction/blob/master/src/Apps/Artist/routes.tsx#L110-L118

and then assumes a `variables` option exists since that's how its seen elsewhere in QueryRenderers, etc.